.github/workflows/run-ci-cd.yaml

```
@@ -209,171 +209,30 @@ jobs:
+      aws_role_session_name: GitHubActions-StagingDeploy
+      backend_use_fargate_spot: true
+      django_configuration: Staging
+      django_settings_module: settings.staging
+      enable_cron_tasks: false
+      enable_nat_gateway: false
+      enable_rds_proxy: false
+      environment: staging
+      frontend_use_fargate_spot: true
+      index_task_definition: nest-staging-index-data
+      migrate_task_definition: nest-staging-migrate
+      release_version: ${{ needs.set-release-version.outputs.release_version }}
```

actionlint (1.7.12)

[error] property "set-release-version" is not defined in object type {bootstrap-nest-staging-infrastructure: {outputs: {}; result: string}}

(expression)